ocaml/xenopsd/lib/xenops_server.ml

```diff
@@ -1763,7 +1763,7 @@ let rec atomics_of_operation = function
           serial "VIF.activate_and_plug" ~id
             [VIF_set_active (vif.Vif.id, true); VIF_plug vif.Vif.id]
       )
-  | VM_restore_devices (id, restore_vifs) ->
+  | VM_restore_devices (id, restore_vifs, migration) ->
```

Comment:
There's no need to add a new boolean, restore_vifs indicates the migration status already (it's false on migration only) It's probably worth defining restore_vifs from the migration: `let restore_vifs = not migration`